internal/core/textlayout/sharedparley.rs

```diff
@@ -550,6 +550,7 @@ pub fn draw_text_input(
         LayoutOptions {
             max_width: Some(width),
             max_height: Some(height),
+            horizontal_align: text_input.horizontal_alignment(),
```

Comment:
Oh no way haha, didn't expect it to be that easy.

I write a `Default` impl for `LayoutOptions` when it was public and meant to be used by all the backends, but now that it's private and just in `sharedparley`, maybe should get rid of the `impl Default` so that we're forced be be more explicit and this doesn't happen again?

Reply:
Did that now, and found a bug on the way :)